Say/ViewController.swift

```diff
@@ -120,37 +120,33 @@ class ViewController: NSViewController {
     
     @IBAction func say(_ sender: NSToolbarItem) {
         
-        //temp = Say(text: self.textForSpeech, voice: self.selectedVoice).play(true)
-        //temp = Say(text: self.textForSpeech,voice: self.selectedVoice)
         if !temp.isplaying(){
             
             if self.pause{
             
                 temp.continueSpeeking()
             }
             else{
-                //sender.isEnabled = false
+                sender.isEnabled = false
                 temp = SayAPI(text: self.textForSpeech, voice: self.selectedVoice)
+                temp.speechToolBar = sender
                 temp.play(false)
                 
             }
             //sender.isEnabled = true
         }
-        else{
-            
-        }
         
     }
     @IBAction func pause(_ sender: NSControl) {
         
-        sender.isEnabled = true
+        //sender.isEnabled = true
```

Comment:
If you don't want to remove unnecessary code, I'd like you to add a comment that Why or Todo something with that code at least. So, It will be help other developers later